code/game/gamemodes/dynamic/dynamic_simulations.dm

```diff
@@ -6,7 +6,6 @@
 
 /datum/dynamic_simulation/proc/initialize_gamemode(forced_threat, roundstart_players)
 	gamemode = new
-	gamemode.roundstart_pop_ready = roundstart_players
```

Comment:
erm, this will break threat generation

Reply:
It won't, this doubles roundstart_pop_ready because pre_init call in gamemode sets this up anyway